scripts/vault.sh

```diff
@@ -9,14 +9,15 @@ source ${SCRIPTS_DIR}/common.sh
 VM_IP=${VM_IP:-127.0.0.1}
 
 KV_APP_NAME=${KV_APP_NAME:-primaza}
-KV_PREFIX=${KV_PREFIX:-kv}
+KV1_PREFIX=${KV1_PREFIX:-kv1}
+KV2_PREFIX=${KV2_PREFIX:-kv2}
```

Comment:
Can we use `secret/data`as KV2_PREFIX default rather than `kv2`?

Reply:
I prefer to keep like this as it simplifies the readability and comparison with kv1 when create a key orcheck ACL. Remark. You can override the env var if you pass it like this to the bash script `KV2_PREFIX=secret ./scripts/vault.sh`